src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureDeleteConfirm/ArchivedFeatureDeleteConfirm.tsx

```diff
@@ -0,0 +1,99 @@
+import { Alert, styled } from '@mui/material';
+import React, { useState } from 'react';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import Input from 'component/common/Input/Input';
+import { IFeatureToggle } from '../../../../../interfaces/featureToggle';
+import { formatUnknownError } from '../../../../../utils/formatUnknownError';
+import { useFeatureArchiveApi } from '../../../../../hooks/api/actions/useFeatureArchiveApi/useReviveFeatureApi';
+import useToast from '../../../../../hooks/useToast';
+
+interface IEnviromentDeleteConfirmProps {
+    deletedFeature?: IFeatureToggle;
+    open: boolean;
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    refetch: () => void;
+}
+
+const StyledDeleteParagraph = styled('p')(({ theme }) => ({
+    marginTop: '2rem',
```

Comment:
We should use `theme.spacing` instead of static values whenever possible. 
See: https://mui.com/material-ui/customization/spacing/

```suggestion
    marginTop: theme.spacing(4),
```

Reply:
Fixed